exercises/05.2-Loop-from-seven-to-twelve/README.md

```diff
@@ -7,6 +7,9 @@ tutorial: https://www.youtube.com/watch?v=6eLXV_IL2m0
 ## 📝 Instructions:
 
 1. Count from `7` to `12` with a loop and print each number on the console.
+## 💡 Hint:
```

Comment:
Add a space or empty line between the instructions and the hints (Between the lines)

Reply:
done